package.json

```diff
@@ -53,7 +53,7 @@
       "git add"
     ],
     "*.md": [
-      "prettier --write",
+      "prettier --single-quote --write",
```

Comment:
Wondering if we should create a `.prettierrc` file for this? Not holding up approval for it, just a thought for future.

Reply:
That'll screw with eslint-config-rollup. I'd like to not use a `.prettierrc` if we can help it.

Reply:
got it.

Reply:
we could also create some config file for this and pass the config file in the run command here if we end up adding more settings here.

Reply:
that'd be an option